persistence/src/main/resources/db/migration/schema/V1_18__AlterCandidatesTable.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE candidates ADD CONSTRAINT uc_cv_file_name UNIQUE (cv_file_name) ;
```

Comment:
This line seems to be redundant if the UNIQUE constraint is added in the V1_2 script.
Also Flyway reports this with the line

    2016-09-12 11:46:30 [localhost-startStop-1] WARN  o.f.c.i.dbsupport.JdbcTemplate - DB: Duplicate index 'uc_cv_file_name' defined on the table 'atsy.candidates'. This is deprecated and will be disallowed in a future release. (SQL State: HY000 - Error Code: 1831)

While the flyway scripts are executed successfully, I can still see a unique index called cv_file_name